config/suppressions.xml

```diff
@@ -93,12 +88,7 @@
   <!-- HandlerFactory crosses allowed limit for executable statements -->
   <suppress checks="ExecutableStatementCount" files="HandlerFactory\.java"/>
 
-  <suppress id="ImportControlTest" files="[\\/]powermock[\\/]"
-       message=".* - org\.(powermock|mockito).*" />
-
   <suppress id="noSourceforgeNetLinks" files="[\\/]releasenotes.xml"/>
-  <suppress id="noSourceforgeIoLinks" files="[\\/]google_style.xml" lines="31"/>
-  <suppress id="noSourceforgeIoLinks" files="[\\/]sun_style.xml" lines="31"/>
```

Comment:
I am not remembering these weird suppressions, but I assume they were temporary for some reason?